app/Console/Commands/InteractsWithCollections.php

```diff
@@ -34,6 +35,7 @@ public function forEachCollection(Closure $callback, Closure $queryCallback = nu
             ->when($queryCallback !== null, $queryCallback)
             ->select('collections.*')
             ->withoutSpamContracts()
+            ->onlyErc721()
```

Comment:
not sure about the `only` prefix, in theory I can add a `->orWhere('type', OTHER)` so is not necessary accurate, thoughts?